.github/workflows/pr_test_peft_backend.yml

```diff
@@ -110,23 +110,23 @@ jobs:
         python -m venv /opt/venv && export PATH="/opt/venv/bin:$PATH"
         python -m pytest -n 4 --max-worker-restart=0 --dist=loadfile \
           -s -v \
-          --make-reports=tests_${{ matrix.config.report }} \
+          --make-reports=tests_${{ matrix.config.report }}_${{ matrix.lib-versions }} \
```

Comment:
I can't see `config`  being inside `matrix`. But it is already on `main`. So if you are sure it works this way, I am fine too.

Reply:
Yeah this part `tests_${{ matrix.config.report }}` has not ever failed. So, probably okay.

Reply:
`config` is not defined, and it's currently empty - we can confirm this with the the artifact name here being `pr__test_reports` 
https://github.com/huggingface/diffusers/actions/runs/10832370808/job/30056608644?pr=9411#step:8:3
It never "failed" before, but failing now is because of the difference between actions/upload-artifact@v2 and v4 https://github.blog/news-insights/product-news/get-started-with-v4-of-github-actions-artifacts/#file-stomping-and-corrupted-artifacts; so I think previously it was just silently overwrote 

so yeah, agree with @ydshieh here, I think it is better we do not keep that part

Reply:
Yeah right. I will work on this PR today to get things to a saner state. Thanks for bearing.

Reply:
Fixed.